devicehive-backend/src/main/resources/hazelcast.xml

```diff
@@ -1,47 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-  #%L
-  DeviceHive Java Server Common business logic
-  %%
-  Copyright (C) 2016 DataArt
-  %%
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
-  #L%
-  -->
-
-<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd"
-           xmlns="http://www.hazelcast.com/schema/config"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-
-    <network>
-        <join>
-            <multicast enabled="false"/>
-            <aws enabled="false"/>
-            <tcp-ip enabled="false"/>
-        </join>
-    </network>
-
-    <map name="NOTIFICATIONS-MAP">
```

Comment:
They are default. We can't change them from client. https://stackoverflow.com/questions/10603254/using-a-maploader-for-a-hazelcast-distributed-map-within-a-client